app/helpers/cms_helper.rb

```diff
@@ -35,4 +35,41 @@ def cms_page_path page
   rescue
     "#"
   end
+
+  def hasParent? page
+    page.parent
+  end
+
+  def breadcrumb_link page
+    "<a href='/c#{page.full_path}' title='Visit #{page.label}' class='breadcrumbs__link'>#{page.label}</a>"
+  end
+
+  def get_breadcrumbs current_page
+    page = current_page
+    breadcrumbs = []
+
+    breadcrumbs.push(breadcrumb_link(page))
```

Comment:
When there is only one argument passed in to the array, I would use `<<` rather than `push`.
So `breadcrumbs << breadcrumb_link(page)`